test/expected/ts_merge-17.out

```diff
@@ -2549,3 +2539,16 @@ DROP USER regress_merge_privs;
 DROP USER regress_merge_no_privs;
 \o
 :DIFF_CMD
+--- Base pg table results
++++ Hyperatable results
+@@ -326,7 +326,9 @@
+ 
+  delta | sid | balance | tid | balance 
+ -------+-----+---------+-----+---------
+-(0 rows)
++   -10 |  -1 |      -1 |  -1 |     -11
```

Comment:
I think this test result is unexpected, no?

Reply:
ts_merge test is huge mess, lots of errors and the non-ht vs ht dont even work with equivalent datasets

Reply:
ts_merge needs another look, there is something wrong with that test that is not pg17 related

Reply:
will be done in followup-pr